test/functional/tests.py

```diff
@@ -174,6 +174,21 @@ def test_get_file(self):
         os.remove(test_hash)
         self.assertNotIn(test_hash, os.listdir(os.getcwd()))
 
+    def test_get_file_with_rename(self):
+        self.api.add(self.fake_file)
+
+        test_hash = self.fake[0]['Hash']
+        local_file_name = "test_file.txt"
+
+        # Download the file and save it to the current working
+        # directory with the given name
+        self.api.get(test_hash, output_path=local_file_name)
+        self.assertIn(local_file_name, os.listdir(os.getcwd()))
+
+        self.assertTrue(local_file_name in os.listdir(os.getcwd()))
```

Comment:
Because I copied the test from the plain `test_get_file` and then didn't see the duplication of logic. Too fixated on checking that the file was in the directory to see that it was already checked for. Nice catch! I'll remove that.